sources/packages/backend/apps/test-db-seeding/src/main.ts

```diff
@@ -0,0 +1,36 @@
+require("../../../env_setup_apps");
+import { NestFactory } from "@nestjs/core";
+import { CleanDb } from "./clean-db/clean-db";
+import { TestDbSeedingModule } from "./test-db-seeding.module";
+import { SeedExecutor } from "./seed-executors/seed-executor";
+// Clean db command line parameter.
+const CLEAN_DB = "--task=clean-db";
+
+async function bootstrap() {
+  const app = await NestFactory.create(TestDbSeedingModule);
+
+  // Checking for CLEAN_DB parameter.
+  if (process.argv.includes(CLEAN_DB) && process.argv.length === 3) {
```

Comment:
Can you elaborate on why do we need the second check (`process.argv.length === 3`)? The first one seems to be enough, doesn't it?

Reply:
It's to verify that the parameter is not passed with the class names to be executed. maybe this will give you more details, https://github.com/bcgov/SIMS/pull/1525#discussion_r1023066516